### ISSUE TITLE
Job kubevirt-io-periodic-link-checker of type periodic ended with state failure

### ISSUE THREAD

**Issue by brybacki** [kubevirt/kubevirt.github.io#613]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
/kind bug
> /kind enhancement

**What happened**:
Periodic job failed 

**What you expected to happen**:

**Anything else we need to know?**:
Log shows:
```
`/` is not writable.
Bundler will use `/tmp/bundler20200812-16-12vvuvj16' as your home directory temporarily.
Could not locate Gemfile
```
> URL where the problem can be found ...
> If the issue is with a lab, please provide information about your environment, platform, versions, ...
Prow link:
https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/logs/kubevirt-io-periodic-link-checker/1293456137366016000

**Comment by mazzystr** [comment]
Prow and Netlify have been causing issues lately...  Prow just had a ruby version update that forced us to update.  Netlify had an outage yesterday.  What pull req was this regarding?

**Comment by mazzystr** [comment]
Are you doing `touch Gemfile.lock` prior to running the container?

**Comment by ksimon1** [comment]
occured today again: https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/logs/kubevirt-io-periodic-link-checker/1295993689977393152

**Comment by mazzystr** [comment]
I understand our CI better now.   I'm investigating.

**Comment by enp0s3** [comment]
@mazzystr happened again today: https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/logs/kubevirt-io-periodic-link-checker/1297806090070659072

**Comment by mazzystr** [comment]
Possibly fixed via https://github.com/kubevirt/project-infra/pull/615

**Comment by codificat** [comment]
> Possibly fixed via [kubevirt/project-infra#615](https://github.com/kubevirt/project-infra/pull/615)

This was actually fixed by this, so we can
/close

this one
